framework/mcp/example_pg_mcp_install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+# Acquire MCP server part of `pg-mcp`.
+# https://github.com/stuzero/pg-mcp
+# https://github.com/crate-workbench/pg-mcp
+
+# FIXME: Improve installation after packaging has been improved.
+#        https://github.com/stuzero/pg-mcp/issues/10
+
+rm -rf pg-mcp
+git clone --depth 1 --no-checkout --filter=blob:none \
+  https://github.com/crate-workbench/pg-mcp.git
+cd pg-mcp
+git checkout 16d7f61d5b3197777293ebae33b519f14a9d6e55 -- pyproject.toml uv.lock server test.py
+uv pip install .
```

Comment:
Dear @stuzero,

thanks for your sweet response, we appreciate it. We see room for improvements here, in order to get rid of this installation obstacle.

- https://github.com/stuzero/pg-mcp/issues/10

We will be happy to help, just let us know.

With kind regards,
Andreas.